lib/chat_widget.dart

```diff
@@ -68,26 +100,52 @@ class _SingleChatState extends State<SingleChat> {
                   color: Color(0xFFE2F0FE),
                   borderRadius: BorderRadius.all(Radius.circular(8)),
                 ),
-                child: Text(widget.chat.text),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(widget.chat.text),
+                    if (imageSelected) const SizedBox(height: 10),
+                    if (imageSelected)
```

Comment:
it works having two if statements that are the same, but I think it would be better to just have one.

A way to accomplish this is with the spread operator.  ie `if (isImageSelected) ...[]` with the spacer and container in here.